packages/toggle/Toggle.test.tsx

```diff
@@ -1,17 +1,153 @@
-import { Toggle } from './Toggle';
-
-import { h, mount } from 'bore';
+import * as expect from 'expect';
+import { h, mount, WrappedNode } from 'bore';
+import { emit } from 'skatejs';
+import { GenericEvents } from '../_helpers';
+import { Toggle } from './index';
 
 describe( Toggle.is, () => {
 
-  it( 'should render', () => {
+  describe( `Custom element`, () => {
+
+    it( `should be registered`, () => {
+
+      expect( customElements.get( Toggle.is ) ).toBe( Toggle );
+
+    } );
+
+    it( `should render via JSX IntrinsicElement`, () => {
+
+      return mount(
+        <bl-toggle />
+      ).wait(( element ) => {
+
+        expect( element.node.localName ).toBe( Toggle.is );
+
+      } );
+
+    } );
+
+    it( `should render via JSX class`, () => {
+
+      return mount(
+        <Toggle />
+      ).wait(( element ) => {
+
+        expect( element.has( '.c-toggle' ) ).toBe( true );
+
+      } );
+
+    } );
+
+  } );
+
+  describe( `API`, () => {
+
+    describe( `[disabled]`, () => {
+
+      it( `should set disabled state`, () => {
+
+        return mount(
+          <bl-toggle disabled />
+        ).wait( checkDisabled );
+
+      } );
+
+      it( `should set disabled state via attribute`, () => {
+
+        return mount(
+          <bl-toggle attrs={{ 'disabled': true }} />
+        ).wait( checkDisabled );
+
+      } );
+
+      function checkDisabled( element: WrappedNode ) {
+        const input: Partial<HTMLInputElement> = element.one( 'input' ).node;
+        expect( input.hasAttribute( 'disabled' ) ).toBe( true );
+      }
+
+    } );
+
+  } );
+
+  describe( `[color]`, () => {
+
+    it( `should set color`, () => {
+
+      return mount(
+        <bl-toggle color="warning" />
+      ).wait( checkColor );
+
+    } );
+
+    it( `should set color via attribute`, () => {
+
+      return mount(
+        <bl-toggle attrs={{ 'color': 'warning' }} />
+      ).wait( checkColor );
 
-    console.warn( 'Missing tests for Toggle!' );
+    } );
 
-    return mount(
-      <Toggle />
-    ).wait();
+    function checkColor( element: WrappedNode ) {
+      const label: Partial<HTMLInputElement> = element.one( 'label' ).node;
+      expect( label.getAttribute( 'class' ) ).toContain( 'warning' );
```

Comment:
just a nitpick. It's better to use appropriate api for specific task, in this case I would prefer `classList`

`expect(label.classList).toContain('warning')`
or 
`expect(label.classList.contains('warning')).toBe(true)`